src/pages/templates/page-sections/pricing/threeTierPricing.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  List,
+  ListIcon,
+  ListItem,
+  Text,
+  VStack,
+  Button,
+  Badge,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { FaCheckCircle, FaTimesCircle } from 'react-icons/fa';
+
+export default function PricingTable() {
+  const borderBg = useColorModeValue('gray.200', 'gray.500');
+  const pricingItemStyles = {
+    py: 16,
+    px: 8,
+    mb: 4,
+    mt: { base: 12, lg: 24 },
+    mx: { base: 'auto', lg: 4 },
+    borderColor: borderBg,
+    borderRadius: 10,
+    borderWidth: '2px',
+    maxW: '350px',
+  };
+  const priceProps = {
+    fontSize: '5xl',
+    fontWeight: 'bold',
+    my: 3,
+  };
+  const priceTypeProps = {
+    fontSize: '2xl',
+    fontWeight: 'semibold',
+    color: 'gray.500',
+  };
+  return (
+    <Box m="0 auto" bg={useColorModeValue('gray.50', 'gray.800')}>
```

Comment:
Please add some `py={12}` here to add some general padding so it looks not so crowded in the iframe preview

Reply:
Noted. I'll make the required changes :D

Reply:
Hey @OmkarK45 are you still on it? If you don't have the time I can also make the changes. Just let me know 😄

Reply:
:) Sorry ... ! I was busy this week I'll push the new changes soon